federation/.env.dist.template

```diff
@@ -0,0 +1,17 @@
+LOG_LEVEL=$LOG_LEVEL
+PORT=$PORT
+GRAPHIQL=$GRAPHIQL
```

Comment:
```suggestion
GRAPHIQL=$GRAPHIQL
```
```suggestion
GRAPHIQL=false
```

Reply:
This will explicitly set the `GRAPHIQL` value to false, which is a very good value for production.
The same thing is done in the backend.
There is never a need to have a playground on a production system.

Reply:
ok, then this is an exception against all other .template definitions.
it is replaced and added a comment too.